.github/workflows/publishVSCode.yml

```diff
@@ -18,52 +18,49 @@ on:
 # The steps.publsh.outputs.result is an indicator that the pulish to the vscode marketplace succeeded.
 # The steps.mergeDevelop.outputs.status is an indicator that all the publish steps completed.
 jobs: 
-  checkout-code: 
```

Comment:
So I dunno why I can't grok this, but each job run on separate VM.  So for each job you have to do a full checkout if you need access to the codebase.  I updated this flow to bring together the pieces required to do the sha update and push into main.